### ISSUE TITLE
[A11Y]:  <amp-img>, <amp-anim> should propagate `aria-*` to the inner `<img>`

### ISSUE THREAD

**Issue by aghassemi** [ampproject/amphtml#5489]
Currently
`<amp-img aria-label="Some Image"  ..` does NOT read "Some Image" when using screen readers the same way `<img aria-label="Some Image" ...` does.

Same is true for other `aria-*` attributes.

We need to propagate all `aria-*` and `role` attributed from the parent `<amp-img>` to `<img>` since the `<img>` is the real component here.

**IMPORTANT**
I believe it is fine to leave the aria attributes on `<amp-img>` and just propagate them to `<img>`, if we realize we need to actually remove the attributes from `<amp-img>` then we have to be careful to update places such as `<amp-image-lightbox>` that look for attributes such as `aria-describedby` on the customer element.

**To be on the safe side**, we can start by only propagating `aria-label`, `aria-labelledby` and `aria-describedby`
